src/Microsoft.ML.AutoML/Tuner/Flow2.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Microsoft.ML.AutoMLService;
+using Microsoft.ML.SearchSpace;
+
+namespace Microsoft.ML.AutoML
+{
+    internal class Flow2
```

Comment:
>Flow2 [](http://example.com/codeflow?start=19&length=5)

I know this isn't a public class, but some comments would be useful here to better understand what this class does. Just the name doesn't tell much, and knowing more will help with supporting it in the future.

Reply:
It's an implementation of an algo in this [paper](https://www.aaai.org/AAAI21Papers/AAAI-10128.WuQ.pdf)

![image](https://user-images.githubusercontent.com/16876986/162271988-468152dd-23bf-485d-80b5-12ffdbedfebf.png)

I can put a url in code to point to that paper, would that be enough